src/Behat/Config/Gherkin/SimpleFilter.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
I don't think this is needed. If you want to create one of the existing filters just use the corresponding class. And any new filters that you add will not be handled, Behat will throw an exception

Reply:
Unless perhaps all previous filters extend this SimpleFilter with most of the functionality delegated to it

Reply:
Yes, I forgot to remove it. I think we can start simple without it at all.

Reply:
I've just removed it.